app/views/projects/_card.html.erb

```diff
@@ -0,0 +1,21 @@
+<div class="card card-sm">
+  <div class="card-body">
+    <div class="row align-items-center">
+      <div class="col-auto">
+        <span class="bg-muted text-white avatar">
+          <%= inline_svg_pack_tag("static/@tabler/icons/icons/#{icon}.svg") %>
```

Comment:
Debatable, de én nem az ikon megnevezését küldeném be, hanem a teljes "static/@tabler/icons/icons/some_icon.svg" -t. Hiába csak a "some_icon" rész változik most, ha bármelyik ikont le kell cserélni az alkalmazásban, vagy megjelenik a @tabler-en kívül másik ikoncsomag, akkor könnyebb lesz kezelni ezeket. Illetve jelenleg ha belenézek az ikon változóba debugging során, akkor szükségem van a használatának a helyére is, hogy értelmet kapjon a változó.